stack.yaml

```diff
@@ -1 +1 @@
-resolver: lts-13.17
+resolver: lts-13.18
```

Comment:
Also remember that `api/run` has a `--resolver lts-13.13` flag.

Reply:
Good catch, I always miss that one. OT: I think we should just create another submodule/package "api-run" that re-uses the toplevel stack to get a rid of this script.